project/src/store/reducer.ts

```diff
@@ -0,0 +1,21 @@
+import {ActionType, CityType} from '../common/const';
+import {State} from '../types/state';
+import {Actions} from '../types/actions';
+
+const initialState = {
+  city: CityType.Paris,
+  offers: [],
+};
+
+const reducer = (state: State = initialState, action: Actions): State => {
+  switch (action.type) {
+    case ActionType.SetCity:
+      return {...state, city: action.payload};
```

Comment:
```suggestion
    case ActionType.SetCity: {
      return {...state, city: action.payload};
      }
```
у кейсах всегда должны быть курли